cf-net/cf-net.c

```diff
@@ -472,7 +472,9 @@ static int CFNetRun(CFNetOptions *opts, char **args, char *hostnames)
     int ret = 0;
     char *hostname = strtok(hosts, ",");
     while (hostname != NULL){
-        CFNetCommandSwitch(opts, hostname, args, cmd);
+        if (CFNetCommandSwitch(opts, hostname, args, cmd) != 0) {
+            ret = -1;
```

Comment:
I definitely think you should add an error log statement here, something like

```
Log(LOG_LEVEL_ERR, "Command %s with %s options and %s args failed for hostname %s", cmd, opts, args, hostname);
```

Reply:
Error is already logged if you look at https://github.com/cfengine/core/pull/5673#issue-2767274192

Reply:
ah, must be inside CFNetCommandSwitch(), yes, inside of CFNetOpenConnection(). Ack. :+1: